### DISCUSSION TITLE
Welcome and please introduce yourself!

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#1] [General]
1. Who you are and something unique about yourself (separate from academics) 
2. what is your current research project is
3. If you have had previous exposure to research, if so what kind do and more precisely what role you play
4. What your 3 things you would like to learn about biomedical research

**Comment by escalexobar** [comment]
1.	Who you are and something unique about yourself (separate from academics) 
My name is Alejandro Escobar in Chicago campus, and something unique about myself is that I’m a nontraditional student with a background in veterinarian medicine as a technician.
2.	Who you are working with this summer and what is your research project is
I am going to be working with Dr. Juan Caicedo-Ramirez, A transplant surgeon from Northwestern that studies transplant disparities within the Hispanic community. 
3.	If you have had previous exposure to research, if so what kind and more precisely what role you play
I did research with HCOE in 2016 SRPonHD with the Sickle cell center. It was laboratory research, and I got to shadow a hematologist. He shared with me how UI Health was the first to cure SCD via bone marrow/stem cell transplant with matching HLA of donor to recipient: typically a sibling. 

I also did research in 2017-2018 right before graduating from UIC with UIC’s ophthalmology department. There I assisted with obtaining and preparing corneal samples for various tests post corneal injury. I was published with this one.
4.	What your 3 things you would like to get out of this 8-week program
1.	Publication
2.	Shadowing experience
3.	Mentor

**Comment by jaq-serrano** [comment]
1. Who you are and something unique about yourself (separate from academics)
 My name is Jaq Serrano (pronouns: they/them), Chicago campus. I was born and raised in Atlixco, Puebla (MX) until I was 9 years old when my family immigrated to the west suburbs of Chicago. 

2. Who you are working with this summer and what is your research project is
I will be working with Dr. Gelila Goba, an OB/GYN physician at UI Health. Dr. Goba's current research focuses on the correlation of structural violence with the persistence and progression to cervical cancer. 
 
3. If you have had previous exposure to research, if so what kind do and more precisely what role you play
I was a research assistant in an organic chemistry lab at Williams College where I assisted in the synthesis, purification, and analysis of drug-delivery polymeric micelles. In 2016, I worked in a stem cell research lab at Rockefeller University where I used CRISPR-Cas9 to insert a GFP gene in the locus of an early hair follicle marker to track the expression of differentiating skin cells. 

4. What your 3 things you would like to learn about biomedical research 
      a) Data cleaning and analysis 
      b) Best sample size for significance in a given context
      c) Addressing confounding factors

**Comment by bguill3** [comment]
1. Who you are and something unique about yourself (separate from academics)
My name is Brenda Guillen and I am on the Chicago campus. I like to sew, and was sewing a lot of masks when the pandemic first started. Unfortunately I have not been able to sew much since medical school started, so I intend on getting back into sewing this summer. 

2. What is your current research project is. 
I am currently working with Dr. Almario from Transplant Surgery at UI Health. The research project is focused on following up with undocumented immigrant patients who received kidney transplants via funds from the Illinois Transplant Fund and verifying their employment and insured status post transplant. 

3. If you have had previous exposure to research, if so what kind do and more precisely what role you play. 
I do not have any prior research experience. 

4. What your 3 things you would like to learn about biomedical research. 
I would like to first off learn as much about research as I can, because I have no prior experience with research. Second, I would like to learn from my peers, and be exposed to other types of research. Third I would like to get a publication if possible.

**Comment by spulido1** [comment]
1. My name is Sonia Pulido and I am on the Peoria campus. My first job ever was a private cello instructor for 4th grade students--I love music and I enjoy working with kids. 
2. I am working with Dr. Nasya Mendoza-Elias and Dr. Paramita Das in the neurosurgery department at University of Chicago. We are studying the management (anesthetic technique and mode of delivery) of Chiari malformation type I in pregnant patients. 
3. During undergrad I was a research assistant in a neuromodulation study at the Shirley Ryan Abilitylab. We paired stimulation technology with strength training exercises in an effort to promote recovery of functionality after spinal cord injury. During my first year of med school, I got involved in two separate studies. For one of them, I am responsible for scheduling and interviewing patients. We are analyzing their speech data, facial expressions, and body movements in order to identify anxiety or depression. The other study is a retrospective analysis that compares two separate neurosurgical procedures (SEEG vs subdural grid electrodes) and we are comparing the severity of post-operative complications such as seizure status after removal of the electrodes from these procedures in patients with refractory epilepsy.  
4. During this summer research program I want to learn as much as possible—study a topic in depth, improve analytical skills, improve scientific writing, improve ability to verbally present & explain things, etc.  I would also like to build and maintain positive connections with mentors in neurosurgery & my peers across the different campuses. And, of course, it would be nice to earn a publication :)

**Comment by CharliePerezSuarez** [comment]
1. Who you are and something unique about yourself (separate from academics)
My name is Charlie Perez-Suarez, an M2 at the Peoria campus. During the last block of M1, I started getting back into running (middle school was the last time I ran long distance). I ran in the Chicago half-marathon about one month ago, and am MAYBE hoping to a full marathon in the fall.

2. what is your current research project is
I am working with UIC's orthopedic surgery department to evaluate Spanish-speaking patient outcomes and whether or not they feel like they better understand pre/post-operative care instructions if they are given videos in Spanish prior to and after surgery.
I am also helping recruit individuals for a study that is looking at the connection between genetic mutations that are associated with a-fib, and then seeing what specific treatments these individuals respond to the best.

3. If you have had previous exposure to research, if so what kind do and more precisely what role you play
I did two years of benchtop research in undergrad, looking at gene expression of genes that code for enzymes that reverse oxidation of the amino acid methionine, as a buildup of proteins with oxidized methionine may be associated with increased risk of Alzheimer's. I really did not enjoy just doing benchtop research, which is why I wanted to become involved in clinical research instead. 

4. What your 3 things you would like to learn about biomedical research
I would love to learn more about the process of going about getting published, what current researchers wish they knew about research before they began, and learning how to discuss my research more professionally, as I know I'll have to discuss it eventually during residency interviews.

**Comment by mizquierdo2** [comment]
Who you are and something unique about yourself (separate from academics)
 - My name is Melanie Izquierdo and I am an M2 at the Chicago campus. When I was born, I was born on the 8th in delivery room 8 weighing 8 lbs and 8 oz! So 8 is definitely my lucky number haha
What is your current research project is
 - I will be working with Project HEAL, Dr. Janet Lin, and Cammeo Manuel-Medici to look at care follow-up after +HCV diagnosis at UI-Health. 
If you have had previous exposure to research, if so what kind do and more precisely what role you play
 - I was a wet lab intern for and Endocrinology, Nutrition, and Metabolism at the University of Chicago Pritzker School of Medicine. 
 - During my gap year, I was a Junior Clinical Data Abstractor for breast, cervical, prostate, and gallbladder cancer for Tempus Labs, a biotechnology company in Chicago. 
What your 3 things you would like to learn about biomedical research
 - The publication process
 - Presentation of research within my colleagues 
 - The role diversity and inclusion currently plays in research

**Comment by Jdlopez0108** [comment]
1. Who you are and something unique about yourself (separate from academics)
My name is Jason Lopez and I'm a Chicago native! I'm big into music. My primary instrument is guitar and I can sing and freestyle. I have a decent size portfolio of covers and I can freestyle about pretty much anything and everything on command. :-)

2. what is your current research project is
My current project looks at the effects of Time Restricted eating compared to Calorie Counting (Standard of Care) and control (obviously), specifically in Type 2 Diabetes patients. 

3. If you have had previous exposure to research, if so what kind do and more precisely what role you play
I did research in Organic Chemistry for about a year and a half in Undergrad. Our goal was to synthesize a novel amino acid (2-methylproline) and I worked with NMR, GCMS, IR , and things like Liquid/column and Thin Layer Chromatography.

4. What your 3 things you would like to learn about biomedical research
- I'm interested in gaining the perspective of a researcher in clinical trials. It's interesting to be working in one and learning to understand the work and the barriers that go into it. Most importantly, I think it's important to understand these things when analyzing other clinical trials. 
- I'm really excited to get to know everyone in the program and form relationships in the small group sessions. I feel like I do particularly well in small groups and I'm excited to make relationships and gather resources from it. 
- Specifically, I want to learn about nutrition and the strategies and key concepts behind teaching nutrition to patients. I feel so lucky to be able to jump on board with my research group because it's something I've been interested in for a long time. I want to be able to take this research and apply it to my future career as a physician. I'm of the opinion that we don't get nearly enough education in nutrition as medical students, so I'm super excited to learn about nutrition in general so that I can efficiently counsel patients in this area.

**Comment by BlakeAcebo** [comment]
1. I am Blake Acebo and I just finished my first year at the Chicago campus! I am from California originally but have been really liking the city so far. I like to write, and I'm hoping to finish (or honestly, start) a new book this summer.

2. My current research project involves the recent Monkey Pox outbreak, the mode of transmission and replication of the virus, and the efficacy of the small pox vaccine as PEP. 

3. Previously, I worked in an Addiction and Neuroscience research lab. I did a lot of bench work and spent a substantial time managing rats. I helped with data collection and analysis, with some experience presenting and writing methods sections.

4.  I would like to learn how to properly cite sources and similar experiments, while also learning how to incorporate active bias checks into projects.

**Comment by jsejo** [comment]
1. My name is Jessica Sejo (she/her) and I am on the Chicago campus. I'm originally from Fort Worth, TX, but have since lived in Austin, Dallas, Palo Alto, and now Chicago! My OG unique fact was that I had contracted COVID-19 twice within a 9 week span, but I have since been corrected that my second illness was not COVID... it was just a cold. So I guess my fun fact is that I'm a worrier. 

2. I am working with Dr. Phoenix Matthews (they/them) on their project, Mi Quit CARE, which looks at smoking cessation efforts at Mile Square.

3. Previously, I worked as a Clinical Research Coordinator for 2 years at the Dallas VA Medical Center. This role included patient recruitment, retention, scheduling patient visits, overseeing IRB submissions, renewals, responding to study queries, and meeting with site monitors and pIs to ensure that study protocols were met.

4. I am hoping to learn about data analysis, the publication process, and different types of clinical research.

**Comment by jasson907** [comment]
1. My name is Jasson Barrios and I am on the Chicago campus. Something interesting about me is that I grew up in Guatemala for the first 7 years of my life and then moved to Bridgeport, Chicago. Something I enjoy to do is going hiking with my two dogs and enjoying the outdoors. 

2. I am working in Dr. Ai's lab and it revolves around the holiday heart syndrome after excessive binge drinking and how that correlates with onset of Atrial fibrillation. 

3. I started working in a lab at Loyola in Dr. Naleway's lab from sophomore - senior year and then progressed to work at Rush as a full time research tech for 2 years. Here, I was able to write an NIH supplemental fellowship grant and learn various wet lab techniques. 

4. 3 things I would like to learn is new lab skills, how to be more organized in research and learn how to check for biases.

**Reply by ADParedes** [reply]
I also live in Bridgeport and commonly visit Bridgeport Coffee House, Jackalope, and Bad owl coffee. You may be seeing each other regularly and never realize it!

**Comment by vgodin4** [comment]
1. Hi all! My name is Viri Godinez and I am coming from the Rockford campus. I lived in Chicago for 4 years during undergrad and I miss it all the time, but Rockford has a lot of nature and I love it! Fun fact about me is that I've been vegan for 6 years, and my lovely mom has made it her mission to help make vegan versions of many Mexican dishes (it's been amazing).

2. My current research process will focus on Black/Hispanic patients who have undergone sleeve gastrectomies and we are studying if/when patients rebound to their baseline. I'm curious to see how their individual nutritional education plays a role into this!  

3. In undergrad, I was on a volunteer research team with minimally invasive spinal surgeries with Rush Ortho. I did a lot of excel work and it did not have much involvement with writing papers/abstracts, so I do feel a lot of want to dip into this. I did see some really cool surgeries though! 

4.  I would like to learn the process of writing and publishing, how to present research, and how to be patient with pursuing research projects.

**Comment by gescal3** [comment]
1. Hi! My name is Gaby Escalante and I am on the Chicago campus. I am originally from Chicago, so very happy I got to stay in the area.   This summer I'm trying to get into running and making the most of Chicago's summer. 
2. This summer I am working with Dr. Almario, a transplant surgeon. My current focus is on assessing the progress of patients after a liver transplant, and wether there is a clinical scale that could be implemented into Epic to help asses their daily progress. 
3. Previously, I had research experience in psychology while in undergrad and during my gap years I worked in a clinical research lab at Shirley Ryan Ability Lab. 
4. I would like to learn more about other types of clinical research, learn how to present research within my colleagues for future opportunities, and more about data analysis.

**Comment by cjuare7** [comment]
1. Hello My name is Chris Juarez. I am a second year from the Peoria campus. Something I enjoy doing is secret shops. Different restaurants and commercial business will pay laymen to shop at their establishment and give feedback in a report. This has allowed me to eat at many different restaurants around Chicago and enjoy newer foods. 

2. I am working with Dr. Chad Purnell at the UIC Craniofacial Center. One project is using a software that generates 3D models from CT scans in order to examine inferior turbinate structure and airway volume in patients with dental deformities. I am also working on a systematic review and meta-analysis looking at the effects of maxillomandibular advancement and/or genioplasty and genioglossus advancement to treat obstructive sleep apnea. The last study I am helping with is with Dr. Cameron Heyd at Mt. Sinai on a hand surgery project.

3. I have had some previous exposure to research but have never been published. I was part of the HCOE summer 2013 research on health disparities. I worked with Dr. David Marquez with his research on physical activity in elderly Latinos around Chicago. During my master's program, I did bench work. The project looked at SA nodal cells of a mouse and the role of IP3 receptors in the generation of the upstroke in pacemaker activity. Unfortunately, COVID cut my time short in the lab. 

4. I really hope to have a better understanding and love of research. Despite my previous experience, I have not been keen on understanding the process and I have not necessarily enjoyed the work of research. 
I hope to form a good relationship with this community and become connected with the people here as well as the lab I work in. 
I also would like to learn something from other people's research since we are all working in very different fields.
